milli/src/index.rs

```diff
@@ -150,6 +154,9 @@ impl Index {
         let word_prefix_position_docids = env.create_database(Some(WORD_PREFIX_POSITION_DOCIDS))?;
         let facet_id_f64_docids = env.create_database(Some(FACET_ID_F64_DOCIDS))?;
         let facet_id_string_docids = env.create_database(Some(FACET_ID_STRING_DOCIDS))?;
+        let facet_id_exists_docids: Database<FieldIdCodec, CboRoaringBitmapCodec> =
+            env.create_database(Some(FACET_ID_EXISTS_DOCIDS))?;
```

Comment:
```suggestion
        let facet_id_exists_docids = env.create_database(Some(FACET_ID_EXISTS_DOCIDS))?;
```

Do we really need to specify the type here?